docs/getting-started/overview.rst

```diff
@@ -1,6 +1,106 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. 
+
+.NET Core
+---------
+.NET Core is a cloud-optimized, cross-platform implementation of the .NET Platform. It currently 
+supports three main operating systems: Linux, Windows and OS X. 
+
+There are several characteristics of .NET Core that make it special:
+
+**Cross-platform support** is the first important feature. For applications, it 
```

Comment:
This one needs to deliver the point more directly and quickly. How about:

> **Cross-platform support** enables developing and hosting apps on a variety of operating systems. It's also easy to move an app from one operating system to another, since the .NET API is a higher-level abstraction.